core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/commands.scala

```diff
@@ -87,3 +87,10 @@ case class WriteStreamOperationStart(
     foreach_batch: Option[StreamingForeachFunction])
     extends LeafNode
     with Command
+
+case class CreateVariable(name: String, dataType: DataType, defaultExpr: Option[Expression], replace: Boolean)
```

Comment:
```suggestion
case class CreateVariable(name: String, dataType: DataType, defaultExpr: Option[Expression] = None, replace: Boolean = false)
```
1. use defaults
2. is it shared between snow & TSQL?